lib/test/Test/Pos/Cbor/CborSpec.hs

```diff
@@ -96,32 +96,32 @@ spec = withDefConfiguration $ \_ -> do
                 describe "Blockchains and blockheaders" $ do
                     modifyMaxSuccess (min 10) $ describe "GenericBlockHeader" $ do
                         describe "GenesisBlockHeader" $ do
-                            binaryTest @BT.GenesisBlockHeader
+                            binaryTest @Block.GenesisBlockHeader
                         describe "MainBlockHeader" $ do
-                            binaryTest @BT.MainBlockHeader
+                            binaryTest @Block.MainBlockHeader
                     describe "GenesisBlockchain" $ do
                         describe "BodyProof" $ do
-                            binaryTest @BT.GenesisExtraHeaderData
-                            binaryTest @BT.GenesisExtraBodyData
-                            binaryTest @(BT.BodyProof BT.GenesisBlockchain)
+                            binaryTest @Block.GenesisExtraHeaderData
+                            binaryTest @Block.GenesisExtraBodyData
+                            binaryTest @(Block.BodyProof Block.GenesisBlockchain)
                         describe "ConsensusData" $ do
-                            binaryTest @(BT.ConsensusData BT.GenesisBlockchain)
+                            binaryTest @(Block.ConsensusData Block.GenesisBlockchain)
                         describe "Body" $ do
-                            binaryTest @(BT.Body BT.GenesisBlockchain)
+                            binaryTest @(Block.Body Block.GenesisBlockchain)
                     describe "MainBlockchain" $ do
                         describe "BodyProof" $ do
-                            binaryTest @(BT.BodyProof BT.MainBlockchain)
+                            binaryTest @(Block.BodyProof Block.MainBlockchain)
                         describe "BlockSignature" $ do
-                            binaryTest @BT.BlockSignature
+                            binaryTest @Block.BlockSignature
                         describe "ConsensusData" $ do
-                            binaryTest @(BT.ConsensusData BT.MainBlockchain)
+                            binaryTest @(Block.ConsensusData Block.MainBlockchain)
                         modifyMaxSuccess (min 10) $ describe "Body" $ do
-                            binaryTest @(BT.Body BT.MainBlockchain)
+                            binaryTest @(Block.Body Block.MainBlockchain)
                         describe "MainToSign" $ do
-                            binaryTest @BT.MainToSign
+                            binaryTest @Block.MainToSign
                         describe "Extra data" $ do
-                            binaryTest @BT.MainExtraHeaderData
-                            binaryTest @BT.MainExtraBodyData
+                            binaryTest @Block.MainExtraHeaderData
+                            binaryTest @Block.MainExtraBodyData
```

Comment:
Maybe I'm missing something, but couldn't the block tests be removed from `lib/` now that they're in `block/`?

Reply:
Yeah, I think that was a rebase merge screwup. Will fix.

Reply:
Ok, will you make modifications to this? If you'll move the tests in a later PR I'll go ahead and approve.

Reply:
And now I remember that these cannot be moved to `core` because blocks depend on things elsewhere in the tree. Sigh!